process_form.go

```diff
@@ -546,3 +540,16 @@ func processForm(modelName string, w http.ResponseWriter, r *http.Request, sessi
 	http.Redirect(w, r, newURL, http.StatusSeeOther)
 	return m
 }
+
+// SaveRecord saves the record,
+// a model is represented by a pointer (see NewModel(arg, true) function)
+func SaveRecord(model reflect.Value) {
+	var saverI saver
+	var ok bool
+	saverI, ok = model.Interface().(saver)
```

Comment:
why not declare vars on a fly, e.g. 
`iSaver, ok `:=`

Reply:
changed